### ISSUE TITLE
Ironberry juice unobtainable?

### ISSUE THREAD

**Issue by LycanBlackpaw** [Darkosto/SevTech-Ages#1618]
## Issue / Bug

So I'm not entirely sure this is a bug or not, since it worked in 3.0.4. I'm playing on a server running SevTech 3.0.6 (the server was updated last night), and I can no longer squash ironberries in the crushing tub for juice. I put a lot of work into making a max quality Iron Wine for absorption hearts, but now that ironberry juice can't be made, the wine is worthless.
## Expected Behavior

Expected behavior is crushing ironberries in the crushing tub when jumping on it. Current behavior is no juice is produced.
## Steps to Reproduce (for bugs)

1. Place Ironberries in crushing tub
2. Jump on crushing tub
3. Nothing happens, no juice is produced

## Context

I can no longer squash ironberries in the crushing tub for juice. I put a lot of work into making a max quality Iron Wine for absorption hearts, but now that ironberry juice can't be made, the wine is worthless.
## Logs

* Client/Server Log:
* Crash Log:

## Client Information

* Modpack Version: SevTech 3.0.6
* Java Version: Java 8 Update 161
* Launcher Used: MultiMC

* Server/LAN/Single Player: Server
* Resourcepack Enabled?: No
* Optifine Installed?: No

## Server Information
* Java Version: Java 8
* Operating System: Windows 10
* Hoster/Hosting Soultion: Custom box (Spookelton server group, hosted by Inrix. Can provide Discord link if asked, since I'm not 100% on this info)
* Sponge (Non-Vanilla Forge) Server?: No

* Additional Content Installed?: FTB Utilities is installed on the server for claims

**Comment by ZephyrWindSpirit** [comment]
Can confirm this in single player as well. Ironberry juice is uncraftable with a crushing tub. 3.0.6

**Comment by artdude543** [comment]
Yeah, we removed the juice all together to prevent the Iron issue. But I'll re-add the part to crushing tub and just keep the iron dust recipe removed.

**Comment by WheresMyMtnDew** [comment]
Glad you are working on this- we were just baffled as to why the ironberries were not crushing for the wine. Will this fix be in the next update?
